### ISSUE TITLE
[Customer Query ] Fix the issue with stylesheet for theme which are being enqueued using buddypress

### ISSUE THREAD

**Issue by kjnanda** [rtMediaWP/rtMedia#1409]
@surajkrsingh Please fix this issue as you have already debugged it. 

```
wp_enqueue_style( 'bp-neavuaue-stylesheet-theme', BP_PLUGIN_URL . 'bp-templates/bp-legacy/css/'. $wp_current_stylesheet . ".min.css" );
wp_enqueue_style( 'bp-neavuaue-stylesheet-buddypress', BP_PLUGIN_URL . 'bp-templates/bp-legacy/css/buddypress.min.css', '' );
```

Along with that, please fix the typo error for `nouveau` words as this has been written as **neavuaue**

Let me know if you have any query.
-----
Note - Update the customer once the fix is added in the release. `2534834`

**Comment by kjnanda** [comment]
@thrijith I have checked added changes and they are reflected. But as in testing point of view, testing not possible. Can you please confirm once that added conditional logic would not create any issue?

**Comment by thrijith** [comment]
Yes, seems fine to me.

**Comment by kjnanda** [comment]
I see changes have been added on RTMedia.php file.

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

Check at this line - https://github.com/rtMediaWP/rtMedia/blob/pre-release/app/main/RTMedia.php#L1348

**Comment by kjnanda** [comment]
The changes have been added and tested with dev branch

**Comment by kjnanda** [comment]
The changes have been added and tested with master branch

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V 4.5.8

**Comment by kjnanda** [comment]
Also, informed the customer regarding the update
